CHANGELOG.md

```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
 new overloads. (Note that the old `spanner::*Options` types have not been
 deprecated as they are still used in the `spanner::Connection` interface.)
 
-### New Libraries
+**OTHER CHANGES**:
+
+* feat(spanner): use prevailing Options in ConnectionImpl ([#8466](https://github.com/googleapis/google-cloud-cpp/pull/8466))
+* feat(spanner): convert ReadOptions/PartitionOptions to Options ([#8448](https://github.com/googleapis/google-cloud-cpp/pull/8448))
+* fix(spanner): instantiate OptionsSpan objects in legacy admin calls ([#8440](https://github.com/googleapis/google-cloud-cpp/pull/8440))
+* feat(spanner): convert QueryOptions to Options in main client API ([#8430](https://github.com/googleapis/google-cloud-cpp/pull/8430))
+* fix(spanner): add Options save/restore to PartialResultSetSource ([#8355](https://github.com/googleapis/google-cloud-cpp/pull/8355))
+* feat(spanner): implement x-goog-user-project, part 2 ([#8332](https://github.com/googleapis/google-cloud-cpp/pull/8332))
+* feat(spanner): support x-goog-user-project ([#8316](https://github.com/googleapis/google-cloud-cpp/pull/8316))
+
+### [Storage](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/storage/README.md)
 
-<!-- TODO: update library count with next release -->
+* feat(storage): support AuthorityOption ([#8462](https://github.com/googleapis/google-cloud-cpp/pull/8462))
+* fix(storage): json["projectTeam"] might be present but null ([#8446](https://github.com/googleapis/google-cloud-cpp/pull/8446))
+* feat(GCS+gRPC): implement `SetBucketIamPolicy()` ([#8431](https://github.com/googleapis/google-cloud-cpp/pull/8431))
```

Comment:
@coryan decided not to include GCS+gRPC stuff in the CHANGELOG. we can delete all of them.

Previous discussion: https://github.com/googleapis/google-cloud-cpp/pull/8208#discussion_r796932188

Reply:
Done.